README.md

```diff
@@ -57,7 +58,9 @@ Go to http://localhost:8004/ and login with admin@liqd.net | password
 
 For a celery worker to pick up tasks you need to make sure that:
 - the redis server is running
+    run: redis-cli ping (it should return: PONG)
 - the celery config parameter "always eager" is disabled (add `CELERY_TASK_ALWAYS_EAGER = False` to your `local.py`)
+    To find local.py run: find . -name "local.py"
```

Comment:
there is no local.py when installing from the repo. We need to create it under adhocracy_plus/config/settings.
Nice to add that the local.py can also have other customised settings such as a postgresql database setting for development.

Also would be good to point here that the celery always eager is disabled so that tests can run the celery tasks from the code that is calling them rather than scheduling them with the redis broker.

And we could end this celery section by pointing to our extensive [celery documentation](./docs/celery.md).
 on line 79 below.

Reply:
I'll make the changes as you point out!

Reply:
thanks. once you make and push your changes, click the refresh button next to my reviewer name